packages/sources/css/presets/tailwind/package.json

```diff
@@ -19,15 +19,19 @@
     "url": "git+https://github.com/Decathlon/vitamin-web.git"
   },
   "license": "Apache-2.0",
-  "main": "./index.js",
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "scripts": {
+    "build": "tsc"
+  },
   "files": [
-    "index.js",
-    "theme.js"
+    "src",
+    "dist",
+    "package.json",
+    "tsconfig.json"
   ],
-  "dependencies": {
-    "@vtmn/css": "^0.84.1",
-    "@vtmn/css-design-tokens": "^0.20.0",
-    "@vtmn/css-utilities": "^0.4.2"
```

Comment:
It resolves the circular dependency yes.
But it's a breaking change for users that didn't think to download these packages when consuming our preset.
Before, they came naturally so they can add what we said in our `README` of this preset, and with your proposal, it will break their imports because these dependencies will not come naturally.
Maybe we can add them in `peerDependencies` but I think nx will still say that it's a circular problem.
The only thing that I think it should work is to do this in `"prebuild"` script. By modifying also the root `package.json` file in order to launch `"prebuild"` script in all packages.

Reply:
you definitely have more context than me there so I'll let you do the required changes regarding the circular deps as I do not have enough knowledge not to break things 👍